src/features/myUserAccess/RolesTable.tsx

```diff
@@ -0,0 +1,402 @@
+import React, { Fragment, Suspense, lazy, useCallback, useEffect, useMemo, useState } from 'react';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import { useIntl } from 'react-intl';
+import { fetchRoleForPrincipal, fetchRoles } from '../../redux/roles/actions';
+import messages from '../../Messages';
+
+import { DataView, DataViewState } from '@patternfly/react-data-view';
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import { DataViewCheckboxFilter, DataViewTextFilter, useDataViewFilters } from '@patternfly/react-data-view';
+import DataViewFilters from '@patternfly/react-data-view/dist/cjs/DataViewFilters';
```

Comment:
There are some mixed cjs/dynamic imports just FYI

Reply:
good catch, it looks like DataViewFilters isn't exported by react-data-view, but it should